voltage/file.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         else:
             raise TypeError("f must be a string or bytes")
 
-        filename = filename if not filename is None else "file"
+        filename = filename if filename is not None else "file"
```

Comment:
also wtf is this

Reply:
`is not` is better than `not x is y`, makes it much more readable